HDWalletKit/Core/Crypto/Encryption/ECDSA.swift

```diff
@@ -24,21 +24,26 @@ public final class ECDSA {
         var pKey = secp256k1_pubkey()
         
         var result = secp256k1_ec_pubkey_create(context, &pKey, prvKey)
-        if (result != 1) {
+        if result != 1 {
```

Comment:
@impul 1? add name for this magic number

Reply:
done